Documentation/security/host-firewall.rst

```diff
@@ -35,7 +35,13 @@ Deploy Cilium release via Helm:
 
 The ``devices`` flag refers to the network devices Cilium is configured on,
 such as ``eth0``. If you omit this option, Cilium auto-detects what interfaces
-the host firewall applies to.
+the host firewall applies to. The resulting interfaces are shown in the
+output of the ``cilium-dbg status`` command:
+
+.. code-block: shell-session
```

Comment:
I had a typo in my earlier suggestion, sorry about that!

```suggestion
.. code-block:: shell-session
```

Reply:
@learnitall fixed. Thanks!